.github/debug.md

```diff
@@ -0,0 +1,47 @@
+# How to Perform Debugging
+
+The below instructions are written for Linux but include a method for using a remote debugger on Windows. Unless otherwise specified, the instructions are meant to be executed on a Linux machine.
+
+1. Clone the code to a directory of your choosing with your favorite git client or `git clone https://github.com/dell/dellemc-openmanage-ansible-modules.git`
```

Comment:
preferably you an remove this step, we are not seeing this cloned repository being used anywhere 
```suggestion

```